config/initializers/simple_form.rb

```diff
@@ -0,0 +1,166 @@
+# Use this setup block to configure all options available in SimpleForm.
+SimpleForm.setup do |config|
```

Comment:
config.use_govuk_elements_form_builder = true 
turns on the gds form builder. ill add it in the next story which involves styling the form further